archetype/src/main/resources/archetype-resources/crawler-conf.yaml

```diff
@@ -90,6 +90,9 @@ config:
    - application/.+powerpoint.*
    - application/.*pdf.*
 
+  # Tika parser configuration file
+  parse.tika.config.file: "tika-config.xml"
```

Comment:
This will be changed so rarely that I wonder whether it should be put in the default config instead. This would help the generated crawler-conf.yaml as minimal as possible.

Reply:
Yes, of course. My thought was that a configuration used in and deployed via a module shouldn't be defined in core.

Reply:
I had exactly the same thought. I don't mind too much, let's leave it where it is. 
Did you run a crawl with it?